internal/turso/utils.go

```diff
@@ -24,10 +24,38 @@ func marshal(data interface{}) (io.Reader, error) {
 	return buf, err
 }
 
+type ErrorResponseDetails struct {
+	Error interface{} `json:"error"`
+	Code  string      `json:"code"`
+}
+
+func GetFeatureError(body []byte, feature string) error {
+	if len(body) == 0 {
+		return nil
+	}
+
+	var errResp ErrorResponseDetails
+	if err := json.Unmarshal(body, &errResp); err != nil {
+		return fmt.Errorf("failed to unmarshal error response: %w", err)
+	}
+
+	if errResp.Code == "feature_not_available_for_starter_plan" {
```

Comment:
let's move away from printing "starter" for users, as we now call it free.

Reply:
Committed your other change but since this is matching the API response, we'll need to update the API (and Dashboard) to reflect those changes too. I'll create a new issue.